pandas/core/arrays/datetimes.py

```diff
@@ -2215,6 +2218,10 @@ def objects_to_datetime64ns(
             return values.view("i8"), tz_parsed
         except (ValueError, TypeError):
             raise err
+    except OverflowError as err:
+        # Exception is raised when a part of date is greater than 32 bit signed int
+        # TODO: Remove this when its fixed in Python
```

Comment:
you can remove this line of the comment, unless you have a definitive link to an actual bug AND it is active.